src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/util/range/RequestSizeHeapRangeConfiguration.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.util.range;
+
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/** RequestSize controller threshold based on heap occupancy percent range */
+public class RequestSizeHeapRangeConfiguration implements RangeConfiguration {
+
+    private static Collection<Range> rangeConfiguration = defaultRangeConfiguration();
```

Comment:
Can we make this final?  `DEFAULT_RANGE_CONFIGURATION` or make it an instance variable? 
Lets try to avoid static non-final variables. That's hard to reason about and hard to test.